evalai/management/commands/seed.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+
+    help = "My custom django command"
```

Comment:
Can we make the description to match exactly what our management command does ?
This actually makes the code more consistent and it helps in the future whenever one is looking at it.